### PR TITLE
Feature/cu 1cb4phx

### PR DESCRIPTION
**Pull request by vchinnakotla**

### CODE REVIEW

**Comment by joseph-wortmann** [comment]
Task linked: [CU-1cb4phx Refactor the ManagedApp to use the new schema](https://app.clickup.com/t/1cb4phx)

File: echostream_managed_app/__init__.py [hunk 1]
```diff
@@ -331,30 +327,35 @@ def __init__(
...
+        private_auth_tokens = (
+            [
                 auth_data["authorizationToken"]
-                for auth_data in await _run_in_executor(
-                    self.ecr_client.get_authorization_token,
-                    registryIds=[registry.split(".")[0] for registry in registries],
+                for auth_data in (
+                    await _run_in_executor(
+                        self.ecr_client.get_authorization_token,
+                        registryIds=[registry.split(".")[0] for registry in registries],
```
Comment (vchinnakotla) [inline echostream_managed_app/__init__.py:345]: Divided in to private and public, as private would be empty. and call boto3 call with empty registry id was giving error

Comment (vchinnakotla) [reply echostream_managed_app/__init__.py:345]: So only get auth token
 if registries
      else []

File: echostream_managed_app/__init__.py [hunk 2]
```diff
@@ -576,9 +586,9 @@ async def start(self) -> None:
                 ]
             )
             # Now list all existing containers, validate and prune
-            nodes_list: list[
-                ManagedNodeContainer
-            ] = await self.docker_client.containers.list_async(all=True)
+            nodes_list: list[ManagedNodeContainer] = self.docker_client.containers.list(
+                all=True
+            )
```
Comment (joseph-wortmann) [inline echostream_managed_app/__init__.py:591]: Why did you go from list_async to list?

**Comment by joseph-wortmann** [comment]
You need to update the setup.cfg, removing `keywords` key, bumping `version`, updating `classifiers` and remove `[install_lib]` (I have no idea what this is or why it is here)

**Pull request merged**